### PR TITLE
(PUP-6258) Add new Authorization OID arc

### PR DESCRIPTION
**Pull request by nathanielksmith**
This new OID arc adds extensions that can be used to safely sign certs
and subsequently write tk-auth rules around them.

### CODE REVIEW

File: spec/unit/ssl/oids_spec.rb [hunk 1]
```diff
@@ -34,6 +34,9 @@
       'pp_apptier'          => "1.3.6.1.4.1.34380.1.1.24",
       'pp_hostname'         => "1.3.6.1.4.1.34380.1.1.25",
       'ppPrivCertExt'       => '1.3.6.1.4.1.34380.1.2',
+      'ppAuthCertExt'       =>  '1.3.6.1.4.1.34380.1.3',
+      'pp_authorization'    =>  '1.3.6.1.4.1.34380.1.3.1',
+      'pp_auth_role'        =>  '1.3.6.1.4.1.34380.1.3.13',
```
Comment (kylog) [inline spec/unit/ssl/oids_spec.rb]: Nit: let's keep these whitespace aligned with the entries above. It's slightly easier on my eyes :)

And nit unrelated to this PR, but the mix of double-quotes and single-quotes in the entries above has no rhyme or reason here AFAICT.

**Comment by kylog** [comment]
Ping @mrzarquon since you've kinda been stewarding these OIDs in the past.

Also @nathanielksmith have you signed the CLA? If not, please do. But if you have, also let me know because the bot might need a kick.

**Comment by nathanielksmith** [comment]
@kylog whitespace/quotes fixed. And this PR came out of discussion with @mrzarquon , fwiw

**Comment by KevinCorcoran** [comment]
👍

**Pull request merged**